pkg/ocm/helpers.go

```diff
@@ -357,7 +358,17 @@ func (c *Client) LinkOrgToRole(orgID string, roleARN string) (bool, error) {
 	return true, nil
 }
 
-func (c *Client) GetLinkedRoles(orgID string) ([]string, error) {
+func (c *Client) GetAccountLinkedRoles(accountID string) ([]string, error) {
+	resp, err := c.ocm.AccountsMgmt().V1().Accounts().Account(accountID).
+		Labels().Labels(USERRoleLabel).Get().Send()
```

Comment:
If the label is hardcoded inside the func I would rename the func to `GetAccountLinkedUserRoles` so it's more verbose.
The other option is to keep the name as is, and pass the label as parameter, to allow it to be more generic, but I think it's obsolete, since we don't have another use case

Reply:
Good point. 
Changed to `GetAccountLinkedUserRoles`, thanks.